grails-app/services/wekb/ExportService.groovy

```diff
@@ -519,6 +528,45 @@ class ExportService {
 
     }
 
+    Map<String,List> exportPackageTippsAsTSVWithSQL(Package pkg, List status) {
+        log.debug("Begin exportPackageTippsAsTSVWithSQL")
+        DataSource dataSource = BeanStore.getDataSource()
+        Sql sql = new Sql(dataSource)
+        Map<String, String> titleHeaders = getTitleHeadersWithSQL()
+        Map<String, List> export = [titleRow: titleHeaders.keySet()]
```

Comment:
would it not be more convenient to use Map<String, Collection> instead?

Reply:
Which ? The First or second?